spec/js/ClickCounter.spec.js

```diff
@@ -3,59 +3,72 @@ describe("Cookie Clicker", () => {
         game = new ClickCounter();
     });
 
-    it('should increment total clicks', () => {
-        game.increment();
-        expect(game.clicks).toBe(1);
-    });
+    describe('Clicking', () => {
```

Comment:
```suggestion
    describe('Clicking the Cookie', () => {
```
To differentiate from the autoclicker suite